src/main/java/games/strategy/engine/lobby/server/db/Database.java

```diff
@@ -123,7 +123,7 @@ private static void ensureDbTablesAreCreated(final Connection conn) {
         if (!existing.contains("TA_USERS")) {
           final Statement s = conn.createStatement();
           s.execute("create table ta_users" + "(" + "userName varchar(40) NOT NULL PRIMARY KEY, "
-              + "password varchar(40) NOT NULL, " + "email varchar(40) NOT NULL, " + "joined timestamp NOT NULL, "
+              + "password varchar(60) NOT NULL, " + "email varchar(40) NOT NULL, " + "joined timestamp NOT NULL, "
```

Comment:
Probably not, but the "updated password" would still be attempted to be stored in the derby DB